### DISCUSSION TITLE
Page Downloads in Production

### DISCUSSION THREAD

**Discussion by RohanDas28** [vercel/next.js#48999] [Help]
### Summary

I am facing a problem with the latest version of NextJS 13. Whenever I deploy my Next13 project to the server, the page downloads a file every time I try to open it. Can anyone help me with this?

**Comment by Dr-Insanity** [comment]
I have the exact same problem with Python Quart applications! I even deployed a mere quart example. I'm fully aware about the topic of this repository. But after establishing the fact I've experienced the exact behavior you described with an entirely different language just hints it might rather be a vercel problem.
